README.md

```diff
@@ -1,2 +1,211 @@
-# testpilot-metrics
-JS library that handles metrics events for Test Pilot experiments
+# `testpilot-metrics`
+
+The `testpilot-metrics` library sends pings to Google Analytics and Mozilla's
+internal metrics pipeline. It is designed for use by Test Pilot experiments.
+
+
+## Installation
+
+`npm install testpilot-metrics`
+
+The only file you need in your build chain is `testpilot-metrics.js`. It has no
+dependencies.
+
+
+## Usage
+
+### Quick start
+
+Here's a simple node-style example:
+
+```js
+
+const Metrics = require('testpilot-metrics');
+
+const { sendEvent } = new Metrics({
+  id: '@my-addon',
+  cid: 'some-non-PII-user-ID',
```

Comment:
Rather than add a uid generator, I'll just file a bug to document suggestions for generating the user id.